docs/EMULATOR.md

```diff
@@ -0,0 +1,236 @@
+# Emulator User Manual {#emulator}
+
+This document describes how to use the Swadge Emulator as a standalone application. For development
+environment setup instructions, see [Configuring a Development Environment](#setup) instead.
+
+## Installing
+
+To install the emulator, go to the project's [releases page][downloads] and download the appropriate
```

Comment:
Is `[releases page][downloads]` a working link? Or just a reference to later [downloads]? Might as well make it link like `[releases page](https://github.com/AEFeinstein/Super-2024-Swadge-FW/releases)`

Reply:
Oh yeah, I expected to have a lot more links in that section so I went with reference-style but now it looks kinda weird with only one link, I'll change that to a regular one. 

Oh and since the releases page doesn't have anything recent, I _think_ we could use this [create release](https://github.com/marketplace/actions/create-release) action to create a release called something like `latest` that we just update every time main updates instead of creating a new release for stuff that's not really a release. The main reason for this is that plain artifact retention isn't forever and you also can't download them without being logged in.